### DISCUSSION TITLE
Does SFTPGO support ARM CPUs?

### DISCUSSION THREAD

**Discussion by ErminMerdanovic** [drakkan/sftpgo#1370] [Q&A] [answered]
Hello everyone, my apologies for bringing up this question, but I've been unable to locate information on whether SFTPGO supports ARM CPUs, specifically the Ampere Altra. Could anyone kindly provide some insight on this matter? Thank you!

**Comment by drakkan** [comment, marked as answer]
yes, it's standard arm64. It should work fine. May I ask about your use case? Thank you

**Reply by ErminMerdanovic** [reply]
just need a small FTP server for a legacy application that doesn't support modern file transfer protocols

**Reply by drakkan** [reply]
Thank you for explaination. Since Azure seems to use Altra, I thought your question was related to SFTPGo offerings on Azure

**Reply by ErminMerdanovic** [reply]
Ah i see, no i will be using Hetzner they also run on Altra
